app/worker.py

```diff
@@ -172,6 +174,6 @@ async def build_response(self, reduced_results):
             print(chat_result)
             return chat_result
 
-    def get_library_hours(self, library_code):
-        # TODO: LibCal integration here
-        return "9:00am - 5:00pm\n"
\ No newline at end of file
+    async def get_library_hours(self):
+        libcal_utils = LibCalUtils()
+        return await LibCalUtils.get_library_hours(libcal_utils)
```

Comment:
I don't think you need to await at the return statement here since the call to get_library_hours is awaited already

Reply:
I tested without, and we do need the await here. Without it we return a coroutine.